books/free-programming-books.md

```diff
@@ -1692,6 +1692,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Introduction to Computer Science "booksite"](https://introcs.cs.princeton.edu/java/cs/) - by Robert Sedgewick and Kevin Wayne (HTML)
 * [Introduction to Computer science using Java](http://www.programmedlessons.org/Java9/index.html) - Bradley Kjell
 * [Introduction to Programming in Java](http://introcs.cs.princeton.edu/java/home/) - Robert Sedgewick and Kevin Wayne
+* [Introduction to Programming Using Java] (https://www.iitk.ac.in/esc101/share/downloads/javanotes5.pdf) - David Eck (Revised Version)
```

Comment:
```suggestion
* [Introduction to Programming Using Java (5th Edition - revised Nov 2007)] (https://www.iitk.ac.in/esc101/share/downloads/javanotes5.pdf) - David J. Eck (PDF)
```
👮 Following guidelines:
- Include edition/older publish date of resources as part of the title.
- Provide support format.
- Maybe search book in official website: http://math.hws.edu/javanotes (5th edition at https://math.hws.edu/eck/cs124/javanotes5/)

😉

Reply:
Hey @davorpa , Thanks for your help but I dont know why the Travis Build Isnt Passing , There is somewhere an Alphabetical Order mistake so I am bit of confused

Reply:
Dont be afraid. It can be changed using suggestions